src/main/scala/sbtbuildinfo/BuildInfo.scala

```diff
@@ -2,14 +2,77 @@ package sbtbuildinfo
 
 import sbt._, Keys._
 
-case class BuildInfoResult(identifier: String, value: Any, typeExpr: TypeExpression)
+case class BuildInfoResult(identifier: String, value: Any, typeExpr: TypeExpression) {
+  def toStringTuple: (String, String) = (identifier, value.toString)
```

Comment:
You're right, it doesn't seem to fit that well here, but `def renderKeys(infoKeysNameAndValues: Seq[BuildInfoResult]): Seq[String]` in `BuildInfoRenderer`. Perhaps just an auxiliary function in the object will do?